test/OpenTelemetry.Tests/Implementation/Trace/Export/BatchingActivityProcessorTests.cs

```diff
@@ -274,21 +277,36 @@ public async Task ShutdownOnNotEmptyQueueFullFlush()
                                         .SetSampler(new AlwaysOnActivitySampler())
                                         .AddProcessorPipeline(pp => pp.AddProcessor(ap => activityProcessor)));
 
-            var activities = new List<Activity>();
-            for (int i = 0; i < 100; i++)
+            using (var testListener = new TestEventListener())
```

Comment:
Probably a minor improvement of readability and save some indentation spaces:

```suggestion
            using var testListener = new TestEventListener();
```